src/components/SearchModal/CurrencyList.tsx

```diff
@@ -1,27 +1,28 @@
 import { Trans } from '@lingui/macro'
 import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { LightGreyCard } from 'components/Card'
+import QuestionHelper from 'components/QuestionHelper'
+import useTheme from 'hooks/useTheme'
 import { CSSProperties, MutableRefObject, useCallback, useMemo } from 'react'
 import { FixedSizeList } from 'react-window'
 import { Text } from 'rebass'
 import styled from 'styled-components/macro'
+
+import TokenListLogo from '../../assets/svg/tokenlist.svg'
+import { useIsUserAddedToken } from '../../hooks/Tokens'
 import { useActiveWeb3React } from '../../hooks/web3'
 import { useCombinedActiveList } from '../../state/lists/hooks'
 import { WrappedTokenInfo } from '../../state/lists/wrappedTokenInfo'
 import { useCurrencyBalance } from '../../state/wallet/hooks'
 import { TYPE } from '../../theme'
-import { useIsUserAddedToken } from '../../hooks/Tokens'
+import { isTokenOnList } from '../../utils'
 import Column from '../Column'
-import { RowFixed, RowBetween } from '../Row'
-import CurrencyLogo from 'components/CurrencyLogo'
+import CurrencyLogo from '../CurrencyLogo'
+import Loader from '../Loader'
+import { RowBetween, RowFixed } from '../Row'
 import { MouseoverTooltip } from '../Tooltip'
+import ImportRow from './ImportRow'
 import { MenuItem } from './styleds'
-import Loader from '../Loader'
-import { isTokenOnList } from '../../utils'
-import ImportRow from 'components/SearchModal/ImportRow'
```

Comment:
I forgot to notice previous files until here, but keep an eye out on the imports.
It might be overwriting our custom changes to make the file moddable.

For example, it's changing `ImportRow` from absolute to relative import.
Likely we changed that to use our custom `ImportRow` implementation.